public/docs/_examples/dependency-injection/dart/lib/app_component.dart

```diff
@@ -17,25 +16,33 @@ import 'providers_component.dart';
 @Component(
     selector: 'my-app',
     template: '''
-    <h1>{{title}}</h1>
-    <my-car></my-car>
-    <my-injectors></my-injectors>
-    <my-tests></my-tests>
-    <h2>User</h2>
-    <p id="user">
-      {{userInfo}}
-      <button (click)=\'nextUser()\'>Next User</button>
-    <p>
-    <my-heroes id="authorized" *ngIf="isAuthorized"></my-heroes>
-    <my-heroes id="unauthorized" *ngIf="!isAuthorized"></my-heroes>
-  ''',
-    directives: const [CarComponent, HeroesComponent, InjectorComponent, ProvidersComponent],
+      <h1>{{title}}</h1>
+      <my-car></my-car>
+      <my-injectors></my-injectors>
+      <my-tests></my-tests>
+      <h2>User</h2>
+      <p id="user">
+        {{userInfo}}
+        <button (click)="nextUser()">Next User</button>
+      <p>
+      <my-heroes id="authorized" *ngIf="isAuthorized"></my-heroes>
+      <my-heroes id="unauthorized" *ngIf="!isAuthorized"></my-heroes>''',
+    directives: const [
+      CarComponent,
+      HeroesComponent,
+      InjectorComponent,
+      ProvidersComponent
+    ],
 // #docregion providers
-    providers: const [Logger, UserService, const Provider(Config, useValue: CONFIG)]
+    providers: const [
+      Logger,
+      UserService,
+      const Provider(Config, useValue: CONFIG)
+    ]
 // #enddocregion providers
-    )
+)
 class AppComponent {
-  UserService _userService;
+  final UserService _userService;
   String title;
```

Comment:
Cleaner to have title final since we should set it only once for the lifetime of the app.